Wikipedia/Code/DonateFunnel.swift

```diff
@@ -275,4 +276,14 @@ import WMF
     func logDonateFormInAppWebViewDidTapReturnButton() {
         logEvent(activeInterface: .webPayProcessed, action: .returnClick)
     }
+
+    func logHiddenBanner(campaignID: String?) {
+        var actionData: [String: String]?
+        if let campaignID {
+            actionData = [:]
+            actionData?["campaign_id"] = campaignID
+        }
+
+        logEvent(activeInterface: .articleBanner, action: .impressionSuppressed, actionData: nil)
```

Comment:
> Is that something we always do, regardless of if it's in the deck or not?

Nope, no need to create an and pass in an actionData dictionary if the deck doesn't mention it for the event. This time it's needed, I noticed `action_data: campaign_id={ID}` wasn't logged in the console because we weren't passing it in here.